wgpu/README.md

```diff
@@ -43,6 +43,12 @@ The following environment variables can be used to configure how the framework e
 
 See [wiki article](https://github.com/gfx-rs/wgpu-rs/wiki/Running-on-the-Web-with-WebGPU-and-WebGL).
 
+## Shaders
```

Comment:
Thank you! Compile time improvements are always good. Should we add a note about `wgsl-out` here? So, people can convert their `GLSL` or `SPV` shaders to `WGSL` at compile time, or in runtime (maybe we need an example for this). Or `naga` can help translate their shaders and they can finish shaders up by hand.

Reply:
love the idea! doing this now